spec/support/feature_helpers.rb

```diff
@@ -330,7 +330,10 @@ def form_is_filled_in_by_form_filler(live_form_link, skip_question: false, confi
       expect(page).to have_content answer_text
     end
 
-    submission_email_reference = find_notification_reference("submission-email-reference")
+    submission_email_reference = nil
+    if ENV.fetch('SES_SUBMISSIONS', false)
+      submission_email_reference = find_notification_reference("submission-email-reference")
+    end
```

Comment:
This can be written as:

```ruby
    submission_email_reference = if ENV.fetch('SES_SUBMISSIONS', false)
                                   find_notification_reference("submission-email-reference")
                                 else
                                   nil
                                 end
```